README.md

```diff
@@ -5,25 +5,37 @@
 <u>To run:</u>
 
 1. Obtain a Census API key [here](https://api.census.gov/data/key_signup.html)
-2. Create a file called `.env` in the root directory. This file is ignored via the .gitignore file to avoid committing secrets.
+2. Create a file called `.env` in the root directory. This file is ignored via the .gitignore file to avoid committing
+   secrets.
 3. Add the following to the `.env`:
+
 ```
 CENSUS_KEY=REPLACE_ME_WITH_CENSUS_API_KEY
 ```
-4. Run scripts:
-   - test_jsons.py: runs all scripts and tests their output
-     - Verifies scripts produce valid jsons
-   - main.py: produces JSON files
 
+4. Run scripts:
+    - test_jsons.py: runs all scripts and tests their output
+        - Verifies scripts produce valid jsons
+    - main.py: produces JSON files
+        - To run with memory profiling on MacOS/Linux:
+            1. Make sure the script is executable: `chmod 755 memory_profiling/run_main.sh`
+            2. Run the script: `./memory_profiling/run_main.sh`
+            3. View the report: `memory_profiling/memory_profile_report.txt`
+        - To run with memory profiling on Windows (TODO need a Windows user to write this):
```

Comment:
There might be more problems for Windows users than I thought. I use Anaconda3, so I'm not sure why PyCharm or other environments are having trouble.